functions/src/days-before-campaign-ends.ts

```diff
@@ -13,5 +13,10 @@ export const daysBeforeCampaignEnds = (
       1
     );
   }
-  return campaignOptions.endOn - now.getUTCDate() + 1;
+  let daysToEnd = campaignOptions.endOn - now.getUTCDate() + 1
```

Comment:
Naming it `daysToEnd` might make the reader forget that it might be negative. I would go as far as naming this `possiblyNegativeDaysBeforeCampaignEnds` to really communicate what this is and to warn the reader that this value is temporary. Notice also how I also reuse the same words as in the function name, to convey the fact that it is the same thing. What do you think?

Reply:
Sure, will do !